earth_enterprise/test.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
This should be moved to a different directory and/or given a more descriptive filename

Reply:
I vote for creating, e.g. a `test` subdirectory under the `scripts` directory to host scripts that are for testing, but are not otherwise required for running Open GEE.